### DISCUSSION TITLE
Auto import vscode

### DISCUSSION THREAD

**Discussion by yudyananda** [chakra-ui/chakra-ui#4608] [Q&A]
Hi, not sure what's wrong but auto import always suggest me the wrong path, anyone here experience the same thing? 

![chakra](https://user-images.githubusercontent.com/29258979/130393470-d486de60-dc0f-4033-ab3b-3d1843214833.jpg)

**Comment by ImBIOS** [comment]
is it helping?

[reference link](https://medium.com/@imamuzzaki/how-to-enable-import-suggestion-in-vscode-for-nextjs-project-4b81f57806aa)

**Reply by yudyananda** [reply]
i'm afraid not

**Comment by viczam** [comment]
This is still happening to me in vscode 1.6. Any idea why?

**Comment by vincaslt** [comment]
I disabled all extensions and I am getting this issue with chakra ui. Seems like a misconfiguration of chakra ui monorepo or vscode not respecting it.

![image](https://user-images.githubusercontent.com/944727/132814449-ed490a1b-4b19-4ef3-a88f-ad1bef18b7ce.png)

![image](https://user-images.githubusercontent.com/944727/132814433-aad2329e-a87b-42ad-a733-7ba82e0c960b.png)

**Comment by IHIutch** [comment]
Bumping this as Im facing the same issue but I'm not sure how to properly report a bug report. Or if this is just a bug with VSCode, see my comment here: https://github.com/microsoft/vscode/issues/132299#issuecomment-920390819

Seems to be working in VSCode 1.59.1, but not the latest, 1.60.1

**Reply by angblab** [reply]
Nope its not a bug with VSCode. I have tried other libraries and its working correctly. The problem is with Chakra-UI it seems.

**Comment by whydnxx** [comment]
Yup, I have the same thing 
![image](https://user-images.githubusercontent.com/50127599/134228228-d0f97951-3f1d-4830-9e17-4b7b87965dc2.png)

**Comment by xddz9** [comment]
Same issue here on the latest version of VSCode/Chakra.

**Comment by jaydenhinrichsen** [comment]
Has anyone found a workaround for this? The only issue related to this that I could find was labeled as `caused-by-extension` which isn't the case for myself and many others. https://github.com/microsoft/vscode/issues/132299

**Reply by yudyananda** [reply]
we have to wait little bit longer until they fix [internal TypeScript](https://github.com/microsoft/vscode/issues/132299#issuecomment-929448434). Thanks to @IHIutch for opening the issue [microsoft/TypeScript#459221](https://github.com/microsoft/TypeScript/issues/45921)

**Reply by corbt** [reply]
According to a a comment on that issue (https://github.com/microsoft/TypeScript/issues/45921#issuecomment-943767005), it seems the VS Code team thinks this is an issue on the Chakra side; specifically they write "@chakra-ui/layout uses an export map and they need to update their package.json to be compatible".

I don't think a fix is incoming from the Typescript team; is there something that Chakra can do?

**Comment by lucasmaffazioli** [comment]
My problem is that VSCode doesn't import at all when i click tab in the suggested Class, which normally would also create an import at the top of the file. I've tested with 0 extensions and no config at all, and was still not able to have auto import ONLY for chakra. I think we should open a issue.

**Reply by IHIutch** [reply]
Good idea, https://github.com/chakra-ui/chakra-ui/issues/5183

**Comment by lucasmaffazioli** [comment]
As commented on the #5183, shorty after upgrading my dependencies and not having any results, new files started to import correctly.

I've upgraded Typescript-dev to 4.5.2, eslint-dev to 8.3.0, and Chakra React to 1.7.2 and it seems that solved the issue for me, but only on new files it seems, maybe my old ones has weird imports. VSCode 1.62.3

**Comment by wlarkspur** [comment]
Sometimes it works after you restart Vscode.
